lib/robots/dor_repo/gis_assembly/normalize_data.rb

```diff
@@ -100,6 +101,18 @@ def output_zip
             @output_zip ||= "#{rootdir}/content/data_EPSG_4326.zip"
           end
 
+          def output_xml_metadata
```

Comment:
In addition to zipping up the XML metadata with the data, retain them outside the zip.